src/uwtools/config/support.py

```diff
@@ -53,6 +55,36 @@ def log_and_error(msg: str) -> Exception:
     return UWConfigError(msg)
 
 
+def represent_namelist(dumper: yaml.Dumper, data: f90nml.Namelist) -> yaml.nodes.MappingNode:
+    """
+    Convert f90 Namelist objects to ordered dicts and serialize.
+
+    :param dumper: The YAML dumper.
+    :param data: The f90nml Namelist to serialize.
+    """
+    # Convert the f90nml Namelist to an ordered dictionary
+    namelist_dict = data.todict()
+
+    # Represent the dictionary as a YAML mapping
+    return dumper.represent_mapping("tag:yaml.org,2002:map", namelist_dict)
+
+
+def represent_ordereddict(dumper: yaml.Dumper, data: OrderedDict) -> yaml.nodes.MappingNode:
+    """
+    Convert ordered dict objects to dictionaries and serialize.
+
+    :param dumper: The YAML dumper.
+    :param data: The ordereddict to serialize.
+    """
+    # Convert the Ordered Dict to a dictionary
+    for key, value in data.items():
+        if isinstance(value, OrderedDict):
+            data[key] = dict(value)
```

Comment:
In this more generalized approach, we will want the recursive version.

Reply:
Strictly, moving it to a representer makes it automatically recursive within YAML, but its just better behavior to loop it here